packages/perseus/src/widgets/explanation.tsx

```diff
@@ -85,9 +94,9 @@ class Explanation extends React.Component<Props, State> {
 
         const caretIcon = this.state.expanded ? caretUp : caretDown;
 
-        const allowTransition = mediaQueryIsMatched(
-            "(prefers-reduced-motion: no-preference)",
-        );
+        const allowTransition =
+            this._mounted &&
+            mediaQueryIsMatched("(prefers-reduced-motion: no-preference)");
```

Comment:
**note:** We don't want animation on by default on initial render in case the user has turned it off (long term, it would be useful if this also could be overridden by webapp profile setting too).

Reply:
I'm confused how this fixes things.  The `mediaQueryIsMatched` function guards against `window.matchMedia` not being a function. Is that helper written wrong or is the fix more about only running on a mounted component?

Reply:
This is a subtle thing, but we don't just want to guard against it being accessed; the initial render on both server and client must be identical, so we don't want to use `matchMedia` at all on first render as the client could give a different rendered result than the server.